lib/node_modules/@stdlib/math/base/special/ceil10/package.json

```
@@ -62,5 +62,84 @@
     "round",
     "nearest",
     "number"
-  ]
+  ],
+  "__stdlib__": {
+    "scaffold": {
+      "$schema": "math/base@v1.0",
+      "base_alias": "ceil10",
+      "alias": "ceil10",
+      "pkg_desc": "round a number to the nearest power of 10 toward positive infinity",
+      "desc": "rounds a number to the nearest power of 10 toward positive infinity",
+      "short_desc": "ceil to power of 10",
```

```suggestion
      "short_desc": "",
```

We can omit this for this package. Although after rereading https://github.com/stdlib-js/stdlib/issues/1147, seems like we may want something. For now, let's omit, and we can clean-up later. See https://github.com/stdlib-js/stdlib/issues/1147 for some additional context regarding fields. That issue is mostly relevant, but may be somewhat dated.